nginx.conf

```diff
@@ -38,33 +38,16 @@ http {
 
     <% require 'json' %>
     <% proxies = JSON.parse(ENV["PROXIES"] || "{}") %>
-    <%
-      def getenv(key)
-        services = JSON.parse(ENV["VCAP_SERVICES"] || "{}")["user-provided"] || []
-        ENV[key] ||
-          services.map { |service|
-            service["credentials"][key]
-          }.find { |value|
-            value
-          }
-      end
-      def string_to_sign(headers)
-        keys = ["$request_method"]
-        keys.concat(headers.map { |header| "$http_#{header}" })
-        keys.concat(["$request_uri", "$request_body"])
-        return keys.join("\\n")
-      end
-    %>
 
     <% proxies.each do |path, route| %>
       location <%= path %> {
         resolver 8.8.8.8;
         set $backend "<%= route %>";
         proxy_pass $backend;
+        proxy_set_header Cache-Control "public, maxage=600";
```

Comment:
I was just double checking this, it looks like we want `max-age` according to the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)